kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: designator-production-app
+  labels:
+    app: designator-production-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: designator-production-app
+  template:
+    metadata:
+      labels:
+        app: designator-production-app
+    spec:
+      containers:
+        - name: designator-production-app
+          image: zooniverse/designator:__IMAGE_TAG__
+          env:
+            - name: MIX_ENV
+              value: prod
+            - name: PORT
+              value: "80"
+            - name: POSTGRES_USER
+              value: cellect_ex
+            - name: POSTGRES_DB
+              value: panoptes
+            - name: POSTGRES_POOL_SIZE
+              value: "2"
+            - name: APPSIGNAL_APP_NAME
```

Comment:
These `APPSIGNAL_*` aren't used in designator, they should be removed. They were on staging template via #98

Reply:
removed here and from the secret as well.